dataeng/jobs/analytics/DBTRun.groovy

```diff
@@ -29,6 +28,7 @@ class DBTManual{
                 stringParam('DBT_RUN_OPTIONS', allVars.get('DBT_RUN_OPTIONS'), 'Additional options to dbt run/test, such as --models for model selection. Details here: https://docs.getdbt.com/docs/model-selection-syntax')
                 stringParam('DBT_RUN_EXCLUDE', allVars.get('DBT_RUN_EXCLUDE'), 'Models to exlude from this run, the default is known incremental models. Leave these if you are not explicitly updating them!')
                 stringParam('NOTIFY', allVars.get('NOTIFY','$PAGER_NOTIFY'), 'Space separated list of emails to send notifications to.')
+                stringParam('JOB_TYPE', allVars.get('JOB_TYPE'), 'Pass parameter value of manual')
```

Comment:
This should always be `manual`, right? If so, I don't think it makes sense to be a user modifiable parameter. Rather, it seems like an environment variable.

Reply:
right, done